HLTrigger/JetMET/plugins/HLTHFJetCleaner.cc

```diff
@@ -0,0 +1,268 @@
+// -*- C++ -*-
+//
+// Package:    HLTrigger/HLTHFJetCleaner
+// Class:      HLTHFJetCleaner
+//
+/**\class HLTHFJetCleaner HLTHFJetCleaner.cc HLTrigger/HLTHFJetCleaner/plugins/HLTHFJetCleaner.cc
+
+ Description: [one line class summary]
+
+ Implementation:
+     [Notes on implementation]
+*/
+//
+// Original Author:  Alp Akpinar
+//         Created:  Wed, 01 Dec 2021 15:47:19 GMT
+//
+//
+
+// system include files
+#include <memory>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/StreamID.h"
+
+#include "DataFormats/JetReco/interface/PFJet.h"
+#include "DataFormats/METReco/interface/CaloMET.h"
+#include "DataFormats/Math/interface/deltaPhi.h"
+
+//
+// class declaration
+//
+
+class HLTHFJetCleaner : public edm::stream::EDProducer<> {
```

Comment:
Make this a `global` module please, rather than a `stream` one. For the syntax, it's just a few changes, and many examples are available in CMSSW, e.g. [this one](https://github.com/cms-sw/cmssw/blob/master/HLTrigger/JetMET/plugins/HLTMultiplicityValueProducer.h).

Reply:
Implemented in [719d5ca](https://github.com/cms-sw/cmssw/pull/37539/commits/719d5ca05f4a004dfd43eeacec5ef94d5985f383)